src/plinth/javascript/global.gleam

```diff
@@ -9,4 +9,4 @@ pub fn decode_uri_component(a: String) -> String
 
 // https://tc39.es/ecma262/multipage/global-object.html#sec-globalthis
 @external(javascript, "../../global_ffi.mjs", "setTimeout")
-pub fn set_timeout(callback: fn(Nil) -> Nil, delay: Int) -> Nil
+pub fn set_timeout(callback: fn() -> anything, delay: Int) -> Nil
```

Comment:
Doesn't the order of args here also need to be swapped?

Reply:
Ooops, sorry. I messed up here.